### DISCUSSION TITLE
gettin info from dynamic website

### DISCUSSION THREAD

**Discussion by saed220** [skrapeit/skrape.it#238] [Q&A]
hello every body .
i wnat to scrap data from table of website which valuse dynamicilay add by javaScript . i saw [this](https://docs.skrape.it/docs/dsl/extract-client-side-rendered-data). but it seems it does not work with version 1.2.2 . i apprecate any help or code snippt abut how to that with skrap it.

**Comment by christian-draeger** [comment]
hey, 
sorry for inconvenience since this certain example in the docs is outdated.
but for now this should guide you to what you looking for: https://github.com/skrapeit/skrape.it/blob/382f21b276be978c47f4ad9c94df9b207cfa295c/integrationtests/src/test/kotlin/DslTest.kt#L766C9-L779C10

**Reply by saed220** [reply]
thank you . i treyed that but unfortunately it returns empty string insted of text which added by Js

**Reply by IsmaelMatiz** [reply]
hey, I got the same trouble, did you solve it? I print the HTML doc and it seems that I receive the html doc before the required elements load in mi case should like a list of div inside the div.episodes-content, from the next link for instance https://jkanime.net/kimetsu-no-yaiba/
